src/test/java/com/easypost/WebhookTest.java

```diff
@@ -102,9 +112,13 @@ public void testAll() throws EasyPostException {
      * @throws EasyPostException when the request fails.
      */
     @Test
-    @Disabled
     public void testUpdate() throws EasyPostException {
-        // Cannot be easily tested - requires a disabled webhook.
+        _vcr.setUpTest("update");
+
+        Webhook webhook = createBasicWebhook();
+
+        webhook.update();
+        // TODO: We should call this something more intuitive in the future, since it doesn't work like the other Update function
```

Comment:
We need to test this, I suggest we test to see if it's an instance of. I understand it's not great, but it's better than nothing. This is one of those unfortunate things we can't really test well.

The TODO can go. Although the naming is kind of weird, it does indeed update it.